argus/util.py

```diff
@@ -242,6 +246,30 @@ def get_certificate():
     return get_resource("certificate")
 
 
+def _get_command_POWRSHELL(command):
```

Comment:
Function names should be lowercase, with words separated by underscores as necessary to improve readability.